README.md

```diff
@@ -76,7 +76,7 @@ Docker images are available on [DockerHub](https://hub.docker.com/r/rocm/composa
     use a semicolon between each; for example, `gfx908;gfx90a;gfx942`.
 
     ```bash
-    cmake                                                                                             \
+    cmake -S .. -B .                                                                                  \
```

Comment:
I think the `..` in the last line is equivalent to `-S ..`, and `-B .` should be the default setting 🤔